primeqa/mrc/README.md

```diff
@@ -249,13 +249,61 @@ Additionally, to specify a MRQA subset e.g. `SQuAD`, `NaturalQuestionsShort`, `T
        --dataset_filter_column_name subset
 ```
 
-Cross domain experiments can be run by running train and eval as separate processes. The
+Cross domain experiments can be run by running train and eval as separate processes.
 
  
 ### Custom Data
 
 Users can also train (fine-tune) and evaluate the MRC model on custom data by providing their own train_file and eval_file. Instructions for getting started are available [here](../../examples/custom_mrc/README.md).
 
+
+### Training with Multiple Datasets
+
+PrimeQA supports the training of MRC model with combination of multiple datasets, which are specified in "--train_fof" argument. Three formats, csv, jsonl and json, can be used to define "--train_file".
+
+In csv format, the columns of each line, separated by space, are for:
+ - HuggingFace dataset name, or path of local data file, or path of dataset processing script (in python);
+ - Dataset config name or data file format;
+ - Sampling rate within range 0.0 to 1.0, e.g. 0.5 means 50% of the examples are randomly selected and used in MRC training;
+ - Preprocessor name.
+
+If column 2 to 4 are not given, default values from input arguments will be used, i.e.:
```

Comment:
How does the user "not give" a column? For example, if they want to provide only a data file and a preprocessor name, how do they do that?